packages/web3-eth2-beaconchain/src/index.ts

```diff
@@ -0,0 +1,11 @@
+import { ETH2Core } from '../../web3-eth2-core/src/index'
+import { IETH2BeaconChain } from '../types/index'
+import { DefaultSchema } from './schema'
+import { IBaseAPISchema } from '../../web3-eth2-core/src/schema'
+
+// @ts-ignore
```

Comment:
What error were you getting?

Reply:
`Class 'ETH2BeaconChain' incorrectly implements interface 'IETH2BeaconChain'.`

This was a bandaid solution for the TODO: `Lodestar types are not properly implemented for API wrapper methods` I mentioned above - I need help with this